ql/time/businessdayconvention.cpp

```diff
@@ -43,6 +43,8 @@ namespace QuantLib {
             return out << "Unadjusted";
           case Nearest:
             return out << "Nearest";
+          case IMMWednesday:
+            return out << "IMMWednesday";
```

Comment:
This is treated as a date-generation rule instead, see https://github.com/lballabio/QuantLib/blob/master/ql/time/dategenerationrule.hpp